packages/core/src/dynamoDb/commands/viewDynamoDbTable.ts

```diff
@@ -0,0 +1,25 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as vscode from 'vscode'
+import { CancellationError } from '../../shared/utilities/timeoutUtils'
+import { DynamoDbTableNode } from '../explorer/dynamoDbTableNode'
+
+export async function prepareDocument(uri: vscode.Uri) {
```

Comment:
I think this same function is also in searchDynamoDbTables as well

Reply:
removed